lib/hiera/puppet_function.rb

```diff
@@ -70,7 +70,7 @@ def lookup(scope, key, default, has_default, override, &default_block)
     lookup_invocation = Puppet::Pops::Lookup::Invocation.new(scope, {}, {})
     adapter = lookup_invocation.lookup_adapter
     lookup_invocation.set_hiera_xxx_call
-    lookup_invocation.set_global_only unless adapter.global_only? || adapter.has_environment_data_provider?(lookup_invocation)
+    lookup_invocation.set_global_only unless adapter.global_only? || adapter.environment_data_provider?(lookup_invocation)
```

Comment:
AFAIK newer versions of rubocop's `NamingPredicateName` allow [`has_`, `is_` ... prefixes by default](https://docs.rubocop.org/rubocop/cops_naming.html#configurable-attributes-12), so I wouldn't think these would need to change?